src/main/java/org/apache/sling/feature/maven/mojos/ApisJarMojo.java

```diff
@@ -110,7 +110,7 @@
 
     private static final String NON_ASCII_PATTERN = "[^\\p{ASCII}]";
 
-    private static final String EMPTY_STRING = "";
+    private static final String EMPTY_STRING = " ";
```

Comment:
Should really be called 'SPACE' as it's not an empty string any more...

Reply:
right! :) I included the modification in the PR, thanks!